ui/src/app/streams/stream-definitions/deployment-properties/deployment-properties.component.ts

```diff
@@ -66,17 +66,17 @@ export class DeploymentPropertiesComponent implements OnInit, OnDestroy {
    */
   ngOnInit() {
     this.subscriptionFeatureInfo = this.sharedAboutService.getFeatureInfo()
-    .pipe(takeUntil(this.ngUnsubscribe$))
-    .subscribe(featureInfo => {
-      this.skipperEnabled = featureInfo.skipperEnabled;
-      if (this.skipperEnabled) {
-        this.subscriptionPlatforms = this.streamsService.platforms()
-        .pipe(takeUntil(this.ngUnsubscribe$))
-        .subscribe((platforms: Platform[]) => {
-          this.platforms = platforms;
-        });
-      }
-    });
+      .pipe(takeUntil(this.ngUnsubscribe$))
+      .subscribe(featureInfo => {
+        this.skipperEnabled = featureInfo.skipperEnabled;
+        if (this.skipperEnabled) {
```

Comment:
In terms of Skipper related properties, It looks like we only allow platform name to be specified via deployment properties. Are the properties such as package version, repo name are planned to be available as next set of features? [This Shell Command](https://github.com/spring-cloud/spring-cloud-dataflow/blob/15095222f7693b04573ae5aae678e0e991367758/spring-cloud-dataflow-shell-core/src/main/java/org/springframework/cloud/dataflow/shell/command/skipper/SkipperStreamCommands.java#L91) contains the available options for Skipper deployment properties.